### DISCUSSION TITLE
Regarding Student Developer Pack & About Course Completion Doubts

### DISCUSSION THREAD

**Discussion by Nilesh-Bari** [github/india#390] [General]
I have claimed the student developer pack but yet i have not received it....
Do we have to complete the entire modules of azure or the ones we like it??????

**Comment by TheSuranaverse** [comment]
A group for all to get notified for any `Official FRT Announcement`, `Discuss & Clear any Doubts`, `Learn & Complete` together.
One-stop place for daily `Events/Workshops` + `Hackathons` + `Internships/Jobs` + `Free Scholarship, Courses, Swags & Goodies` + `24/7 Live Chat and Mentors to help you with your Doubts`  + `24/7 Live Music` & much more...

Currently, we are helping all the `Future Ready Talent` Interns clear their doubts & queries on a personal basis either on chat or voice, whichever they prefer. We will also be assisting them on Projects & help them complete this internship 🤗

Link to join: https://discord.gg/drQ7ySCdX2
GitHub Solution Forum: https://github.com/github/india/discussions/353

Thank you 🙏

**Reply by ghost** [reply]
I am also waiting for the student developer pack. But till now there is no update on it. If anyone received it please inform all the interns.

**Reply by MOHAMMED-USMAN1** [reply]
hey have you completed the azure fundamentals part 1 all 3 moules if yes plz say me how to complete it

**Reply by rohitsasi720** [reply]
Click on self-learning and start it from there, you will be redirected to Microsoft learning which will record all your progress and will update in the future-ready talent progress bar weekly.

**Reply by TheSuranaverse** [reply]
@prudhvi2001-bot @MOHAMMED-USMAN1 just join the community & ping us there & we will help you
